spec/index.bs

```diff
@@ -1756,7 +1756,7 @@ allows additional usages of WebGPU that would have otherwise been invalid.
 
 <script type=idl>
 enum GPUFeatureName {
-    "depth-clamping",
+    "disable-depth-clip",
```

Comment:
I think this is a poor name for a feature. First, because features are naturally expected to "enable" more function, not disable it.
How about "depth-clip-mode"?

Reply:
"unclipped-depth"?

Reply:
That's better. I think it's also problematic though, since it implies that `depth` isn't processed at all (i.e. doesn't imply clamping).

Reply:
editors: like "depth-clip-control"